src/main/java/org/fundacionjala/virtualassistant/repository/mongo/RecordingRepositoryImpl.java

```diff
@@ -0,0 +1,68 @@
+package org.fundacionjala.virtualassistant.repository.mongo;
+
+import org.bson.Document;
+import org.fundacionjala.virtualassistant.models.mongo.Recording;
+import org.fundacionjala.virtualassistant.repository.mongo.RecordingRepo;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.mongodb.core.MongoTemplate;
+import org.springframework.data.mongodb.core.query.Criteria;
+import org.springframework.data.mongodb.core.query.Query;
+import org.springframework.stereotype.Repository;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.util.Base64;
+import java.util.List;
+
+@Repository
+public class RecordingRepositoryImpl implements RecordingRepo {
+
+    @Autowired
+    MongoTemplate mongoTemplate;
+
+    @Override
+    public Recording getRecording(String idRecording) {
+        Query query = new Query(Criteria.where("idRecording").is(idRecording));
+        Recording document = mongoTemplate.findOne(query, Recording.class);
+        return document;
+    }
+
+    @Override
+    public boolean deleteRecording(String idRecording) {
+        Query query = new Query(Criteria.where("idRecording").is(idRecording));
+        Recording recordingToDelete = mongoTemplate.findOne(query, Recording.class);
+        if (recordingToDelete != null) {
+            mongoTemplate.remove(query, Recording.class);
+            return true;
+        }
+        return false;
+    }
+
+    @Override
+    public List<Recording> getAllRecordingsToUser(Long idUser, Long idChat) {
+        Query query = new Query(Criteria.where("idUser").is(idUser).and("idChat").is(idChat));
```

Comment:
same here, apply DRY concept https://www.baeldung.com/cs/dry-software-design-principle, and use constants